tests/testthat/test-default-methods.R

```diff
@@ -2,6 +2,13 @@ context("default methods")
 set.seed(1111)
 x <- matrix(rnorm(150), 50, 3)
 y <- rnorm(ncol(x))
+lw <- matrix(rnorm(150), 50, 3)
+lw <- sweep(
+  lw,
+  MARGIN = 2,
+  STATS = matrixStats::colLogSumExps(lw),
```

Comment:
We don't have a dependency on matrixStats in rstantools. Can you change this to just do the column log sum exps using base R?

Reply:
I think this should fix the R cmd check issue